test/fixtures/github_repos/module_test_repo.tf

```diff
@@ -5,6 +5,7 @@ module "module-test-repo" {
   description     = "This repository was created by the terraform github repository unit test"
   homepage_url    = "https://github.com/ministryofjustice/terraform-github-repository"
   has_discussions = true
+  poc = false
```

Comment:
terraform fmt

Reply:
Ran that command, didn't pick it up for some reason will fix it